test/integration/servers/graphql-main-server/galoy-pay-testing.spec.ts

```diff
@@ -15,7 +15,7 @@ import { ApolloClient, NormalizedCacheObject } from "@apollo/client/core"
 import { startServer, killServer } from "test/helpers/integration-server"
 
 jest.mock("@services/twilio", () => require("test/mocks/twilio"))
-jest.setTimeout(100 * 1000)
+jest.setTimeout(5 * 60 * 1000)
```

Comment:
this test may need 5 min to complete?

Reply:
The tests wont take that long to complete, but on my local machine it takes about 30 seconds to start up the server. The server seemed to not be starting in the same amount of time in the CI pipeline, so I tried aggressively increasing the timeout. This appears to not have helped and we must be running into some other issue.